src/pages/Main/index.js

```diff
@@ -1,42 +1,91 @@
 import React from 'react';
 import Form from '../../components/Form';
-import List from '../../components/List';
+import ListAlta from '../../components/ListAlta';
+import ListMedia from '../../components/ListMedia';
+import ListBaixa from '../../components/ListBaixa';
 
 class Main extends React.Component {
   constructor(props){
     super(props);
 
     this.state = {
-      data: [],
+      data: [{"Alta": [], "Media": [], "Baixa": []}],
```

Comment:
Por ser um objeto, e não um JSON, vocês poderiam escrever assim: 
`
this.state = {
   data: {alta: [ ], baixa: [ ], media: [] }
}
`